lib/protocol/ResultSet.js

```diff
@@ -150,6 +150,15 @@ ResultSet.prototype.getLobColumnNames = function getLobColumnNames() {
   return this.metadata.filter(isLob).map(getColumName);
 };
 
+ResultSet.prototype.getLobColumnIndexes = function getLobColumnIndexes() {
+  return this.metadata.reduce(function(p, c, i) {
+    if (isLob(c)) {
+      p.push(i);
+    }
+    return p;
+  }, [])
```

Comment:
Something simpler like:

```js
var indexes = [];
for (var idx in this.metadata) {
  isLob(this.metadata[idx]) && indexes.push(idx);
}
return indexes;
```

should work as well. This would be simpler, because in your case with `reduce` the accumulated value is also an array.